wagtail/contrib/search_promotions/forms.py

```diff
@@ -45,12 +67,31 @@ class Meta:
 
 
 class SearchPromotionsFormSet(SearchPromotionsFormSetBase):
+    """
+    Form set for search promotions
+
+    Attributes
+
+        - minimum_forms (int): Number of forms required for this set.
+    """
+
+    REQUIRED_FIELDS_MISSING = _(
+        "You must recommend a related page OR an external link."
+    )
+    TOO_MANY_FIELDS = _("Please only select a Page OR enter an external link.")
+
     minimum_forms = 1
     minimum_forms_message = _(
         "Please specify at least one recommendation for this search term."
     )
 
     def add_fields(self, form, *args, **kwargs):
+        """
+        Args
+
+            - form (forms.Form): Form to add to form set
+        """
```

Comment:
Would suggest removing this. Since `add_fields` is part of the formset API there’s little value in documenting this here.

Reply:
As per this comment, please remove this docstring addition.